package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "crowi",
-  "version": "1.6.4",
+  "version": "1.7.0-dev",
```

Comment:
@hiroppy  how do you think about this versioning

Reply:
I like `rc` but it is 👍 because this is not mejor version:)
FYI: [react@16.3.0-rc.0](https://github.com/facebook/react/releases/tag/v16.3.0-rc.0), [webpack@v3.0.0-rc.0](https://github.com/webpack/webpack/releases/tag/v3.0.0-rc.0)